packages/material-ui/src/AppBar/AppBar.test.js

```diff
@@ -1,23 +1,23 @@
 import * as React from 'react';
 import { expect } from 'chai';
-import { getClasses, createMount, createClientRender, describeConformance } from 'test/utils';
+import { createMount, createClientRender, describeConformanceV5 } from 'test/utils';
 import AppBar from './AppBar';
+import classes from './appBarClasses';
 import Paper from '../Paper';
 
 describe('<AppBar />', () => {
   const mount = createMount();
-  let classes;
   const render = createClientRender();
-  before(() => {
-    classes = getClasses(<AppBar>Hello World</AppBar>);
-  });
 
-  describeConformance(<AppBar>Conformance?</AppBar>, () => ({
+  describeConformanceV5(<AppBar>Conformance?</AppBar>, () => ({
     classes,
     inheritComponent: Paper,
     mount,
+    muiName: 'MuiAppBar',
     refInstanceof: window.HTMLElement,
-    skip: ['componentProp'],
+    testVariantProps: { disabled: true },
```

Comment:
```suggestion
    testVariantProps: { position: 'relative' },
```

Let's use some prop that actually exists ont he component

Reply:
testVariantProps doesn't make sense here because AppBar does not have a variant property. Or is it intended for over reasons?

Reply:
Nevermind just saw the code in a test, its not a variant prop, but a combination of a couple of props in order to apply those.

Reply:
This is related to the variants inside the theme - https://next.material-ui.com/customization/theme-components/#adding-new-component-variants it is not related only to the variant prop, it can be used with any props combination. It is basically allowing developers to add style overrides for some combination of props